tslint.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
`npm run lint` produces `no-unused-variable is deprecated` warning. Maybe related to: [https://github.com/palantir/tslint/issues/4046](https://github.com/palantir/tslint/issues/4046)

Reply:
yes, it should be fixed by the next version of tslint